File: An2/MAP/MAP-Java/MAP-Lab6(NB)/src/Utils/LinkedList/Node.java
/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */
package Utils.LinkedList;

/**
 *
 * @author mihai
 */
public class Node<T> {
    
    public T data;
    public Node<T> next;
    
    public Node(T data){
        this.data = data;
    }
}

File: An2/Sem1/MAP/MAP-C#/MAP-Lab7/MAP-Lab7/Utils/LinkedList/Node.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MAP_Lab7.Utils.LinkedList
{
    public class Node<T>
    {
        public T data;
        public Node<T> next;

        public Node(T data)
        {
            this.data = data;
        }
    }
}
